bin.js

```diff
@@ -61,7 +61,7 @@ var argv = minimist(process.argv.slice(2), {
   if (argv.help) {
     console.log(USAGE)
   } else if (argv.version) {
-    console.log(require('../package.json').version)
+    console.log(require('./package.json').version)
```

Comment:
Certainly! It read as `require('./package.json./package.json')` when I was reviewing earlier, not sure if that was a GitHub error or something else. Looks good now!